### DISCUSSION TITLE
Wagtail 6.0 released (happy 10th birthday Wagtail!)

### DISCUSSION THREAD

**Discussion by gasman** [wagtail/wagtail#11611] [Announcements]
**Today is Wagtail's 10th anniversary! 🎉** To celebrate, we are proud to present Wagtail 6.0.

New features in this release include:
* **Universal listings** - an improved navigation interface for pages and snippets, with built-in searching and filtering
* **A new tutorial**, building on the starting point from "Your first Wagtail site" to the creation of a full-fledged portfolio website
* **Page types usage report**, showing a breakdown of page types used across your site
* **Right-to-left language support** in the admin interface
* **Numerous accessibility improvements**
* **Django 5.0 support**

For the full rundown of new features, enhancements and fixes, see the release notes: https://docs.wagtail.org/en/stable/releases/6.0.html . With this being a major version, various deprecations have taken place, so be sure to check the upgrade considerations.

For general notes on upgrading, see: https://docs.wagtail.org/en/stable/releases/upgrading.html

**Comment by arshyia3000** [comment]
Congrats 
Kudos all for this great masterpiece

**Comment by dostogircse171** [comment]
Yay! Happy Birthday, **Wagtail**! Here's to many more years to come. **Happy coding!**

**Comment by thibaudcolas** [comment]
Happy birthday Wagtail!

<img width="531" alt="Celebrating 10 years of Wagtail" src="https://github.com/wagtail/wagtail/assets/877585/d10b2538-bed6-4980-a0ea-5d9790bf119d">

**Comment by stuaxo** [comment]
Congratulations all round :)

The increase in productivity I felt when I first used Wagtail, felt comparable to the bump when I first used Django years before.

**Comment by oabuhamdan** [comment]
Congrats!
